src/Backend/Modules/Pages/Domain/Page/PageVersionDataGrid.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Backend\Modules\Pages\Domain\Page;
+
+use Backend\Core\Engine\Authentication;
+use Backend\Core\Engine\DataGridDatabase;
+use Backend\Core\Engine\DataGridFunctions;
+use Backend\Core\Engine\Model;
+use Backend\Core\Language\Language;
+
+class PageVersionDataGrid extends DataGridDatabase
+{
+    public function __construct(Page $page, Status $status)
+    {
+        $whereStatement = 'p.id = :id AND p.status = :status AND p.locale = :locale';
+        $parameters =             [
+            'id' => $page->getId(),
+            'status' => $status,
+            'locale' => $page->getLocale(),
+        ];
+
+        if ($status->isDraft()) {
+            $whereStatement .= ' AND p.user_id = :authenticatedUserId';
+            $parameters['authenticatedUserId'] = Authentication::getUser()->getUserId();
+        }
+
+        parent::__construct(
+            'SELECT p.id, p.revision_id, p.title, UNIX_TIMESTAMP(p.edited_on) AS edited_on, p.user_id
+             FROM PagesPage AS p
+             WHERE ' . $whereStatement . '
+             ORDER BY p.edited_on DESC',
+            $parameters
+        );
```

Comment:
Would string interpolation work here?

```
        parent::__construct(
            "SELECT p.id, p.revision_id, p.title, UNIX_TIMESTAMP(p.edited_on) AS edited_on, p.user_id
             FROM PagesPage AS p
             WHERE $whereStatement
             ORDER BY p.edited_on DESC",
            $parameters
        );
```

Reply:
it would, but I find it easier to miss that it is useing that parameter